config/locales/cy.yml

```diff
@@ -36,6 +36,7 @@ cy:
     good_friday:   "Dydd Gwener y Groglith"
     easter_monday: "Dydd Llun y Pasg"
     early_may:     "Gŵyl Banc Cyntaf Mai"
+    early_may_ve:  "Gŵyl Banc Cyntaf Mai (VE day)"
```

Comment:
Does VE day need to be translated to Welsh here?

Reply:
I noted that in the PR comment - 

> We have no translation for Welsh yet so I've left it as "VE day".

Reply:
Probably not that important given the urgency...